UIforETW/Utility.cpp

```diff
@@ -431,108 +859,116 @@ bool Is64BitWindows()
 #else
 	// http://blogs.msdn.com/b/oldnewthing/archive/2005/02/01/364563.aspx
 	BOOL f64 = FALSE;
-	bool bIsWin64 = IsWow64Process(GetCurrentProcess(), &f64) && f64;
+	const bool bIsWin64 = ( ::IsWow64Process(::GetCurrentProcess(), &f64) && f64 );

```

Comment:
A couple of reasons, which are probably moot:

1. (first and foremost is that) I consciously treat any multi-operator expression as suspicious. If it's not totally obvious, I'll always look up [C++ operator precedence](http://en.cppreference.com/w/cpp/language/operator_precedence), to be sure. Adding parenthesis makes precedence explicit, and eliminates any concern for the [insidiousness of C](http://www.underhanded-c.org/_page_id_2.html).
2. The historical basis for the first reason is in my copy of MISRA C (section 8, rule 12.1), which gives as an Advisory guideline ("should be followed as far as is reasonably practical") that expressions containing any of most operators (I actually am unable to copy from the PDF, so I'm summarizing; basically everything from ternary `?:` to cast) should be made explicit.

***That said***, I should really change it to:

```
	return ::IsWow64Process(::GetCurrentProcess(), &f64) && f64;
```

...which looks less awkward anyways. Nothing can go wrong, since [the `return` statement beats everything](https://en.wikipedia.org/wiki/Comma_operator#Examples).